### PR TITLE
* More straight project structure, including template for packaging for ...

### PR DESCRIPTION
**Pull request by hotsyk**
...PyPI.

* Added docs, BSD license.
* Added honcho to emulate Heroku flow while local dev. Try 'honcho start' from root folder.
* Tox is not fully operational yet.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from goodpackagenamehere.control import cli
+from ner_trainer.control import cli
```
Comment (mrgambal) [inline manage.py]: We've decided not to use 'ner_trainer' anymore. 'Goodpackagename' was just like an stub before we come up with a brand new name for that stuff. As @dchaplinsky thought: it should have been a motivation to anybody who will change it.
(not an objection, just an eat for thoughts)

File: AUTHORS.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Development Lead
+----------------
+
+* Mr Gambal <mr_gambal@outlook.com>
+* Dmitry Chaplinsky <chaplinsky.dmitry@gmail.com>
+
+Contributors
+------------
+
+* Volodymyr Hotsyk <gotsyk@gmail.com>
```
Comment (mrgambal) [inline AUTHORS.rst]: Don't hesitate to add yourself to authors list. We're completely equal in that good intention.

File: manage.py (hunk 1 above)
Comment (hotsyk) [inline manage.py]: Yep, actually I've changed to ner_trainer also as temporary name, but which looks more production-ready comparing to goodpackagenamehere :) So, if there are decision already on name, I will change to new name, could also change to goodpackagenamehere if needed

Comment (mrgambal) [inline manage.py]: Should we open a new crowd-sourcing project to select a name?))

**Comment by mrgambal** [comment]
:+1:

**Comment by mrgambal** [comment]
@dchaplinsky wouldn't you add some hatespeech in there, would you?

**Comment by dchaplinsky** [comment]
I definitely will

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+
+python:
+  - "3.4"
```
Comment (dchaplinsky) [inline .travis.yml]: Bold statement!

File: AUTHORS.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Mr Gambal <mr_gambal@outlook.com>
+* Dmitry Chaplinsky <chaplinsky.dmitry@gmail.com>
```
Comment (dchaplinsky) [inline AUTHORS.rst]: Add yourself here.

**Comment by dchaplinsky** [comment]
Great.

We still need to decide on the name/packagename.

**Pull request merged**